.circleci/config.yml

```diff
@@ -0,0 +1,75 @@
+version: 2.1 
+
+executors:
+  default:
+    working_directory: ~/repo
+    description: The official CircleCI Ruby Docker image
+    docker:
+      - image: circleci/ruby:2.7.2
```

Comment:
In .ruby-version you set 2.7.4. I guess we should have the same version in CI config.